### PR TITLE
Python 2/3

### PR DESCRIPTION
**Pull request by cdeil**
Some simple edits to make code work on Python 3 (no changes for Python 2).

### CODE REVIEW

**Comment by paulray** [comment]
Looks good, but I don't understand why Python 3 took away the `file()` initializer for a file object. Seems like a strange thing to get rid of.

File: examples/example_B1855.py [hunk 1]
```diff
@@ -15,10 +13,10 @@
 timfile = os.path.join(datadir, 'B1855+09_NANOGrav_dfg+12.tim')
 
 # libstempo calculation
-print "tempo2 calculation"
+print("tempo2 calculation")
```
Comment (paulray) [inline examples/example_B1855.py]: How does this work without an import of `print_function` in Python 2?

**Comment by cdeil** [comment]
> I don't understand why Python 3 took away the file() initializer for a file object. 

I think it was just part of cleanup to get closer to the goal in the [zen of Python](https://www.python.org/dev/peps/pep-0020/#id3)

> There should be one-- and preferably only one --obvious way to do it.

They took away a few things that are duplicated (in this case `file` and `open`) and unneeded
(see https://docs.python.org/3.0/whatsnew/3.0.html#builtins), causing some pain now, but making Python a better language for the long-term future.

File: examples/example_B1855.py (hunk 1 above)
Comment (cdeil) [reply examples/example_B1855.py]: In Python 2, what happens is that the `()` are ignored as a redundant set of parentheses:
```
$ python2
>>> print('spam')
spam
>>> print('spam', 'ham')
('spam', 'ham')
>>> print('spam', )
('spam',)
>>> ('spam')
'spam'
```
I can add the `from __future__ import print_function` line at the top if you want.

**Pull request merged**